refactron/core/refactron.py

```
@@ -228,3 +239,102 @@ def _should_exclude(self, path: Path) -> bool:
+            target: Path to file to analyze
+
+        Returns:
+            Dictionary with AI suggestions
+
+        Raises:
+            RuntimeError: If AI features are not enabled
+        """
+        if not self.ai_service or not self.ai_service.is_enabled():
+            raise RuntimeError(
+                "AI features not enabled. Initialize Refactron with ai_config parameter."
+            )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)